package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "dev": "next dev",
     "build": "next build",
-    "prebuild": "yarn update-approved-identifiers",
+    "predev": "yarn update-approved-identifiers",
```

Comment:
running this before dev instead of build because its extremely fast and this way we'll never have out of date data